transform/mattermost-analytics/models/intermediate/sales/int_cloud_trial_requests.sql

```diff
@@ -0,0 +1,67 @@
+{{
+    config({
+        "materialized": "table"
+    })
+}}
```

Comment:
Does this test really require `table` materialization? Default is ephemeral. I understand it might be useful during debugging, but not sure about production.

Reply:
It was for tests, forgot to remove it.